### PR TITLE
feat(LIVE-14917): add dynamic naming on nano S error

### PR DESCRIPTION
**Pull request by lpaquet-ledger**
### ✅ Checklist

- [x] `npx changeset` was attached.
- [ ] **Covered by automatic tests.** 
- [ ] **Impact of the changes:** 
  - ...

### 📝 Description

add dynamic naming on nano S error

### ❓ Context

https://ledgerhq.atlassian.net/browse/LIVE-14917

---

### 🧐 Checklist for the PR Reviewers

- **The code aligns with the requirements** described in the linked JIRA or GitHub issue.
- **The PR description clearly documents the changes** made and explains any technical trade-offs or design decisions.
- **There are no undocumented trade-offs**, technical debt, or maintainability issues.
- **The PR has been tested** thoroughly, and any potential edge cases have been considered and handled.
- **Any new dependencies** have been justified and documented.
- **Performance** considerations have been taken into account. (changes have been profiled or benchmarked if necessary)

### CODE REVIEW

File: apps/ledger-live-desktop/src/renderer/components/DeviceAction/index.tsx [hunk 1]
```diff
@@ -340,8 +340,11 @@ export const DeviceActionDefaultRendering = <R, H extends States, P>({
     }
   }
 
-  if (device?.modelId === "nanoS" && (request as SwapRequest)?.provider === "thorswap") {
-    return renderHardwareUpdate();
+  if (
```
Comment (jiyuzhuang) [inline apps/ledger-live-desktop/src/renderer/components/DeviceAction/index.tsx:343]: You want to say the swap request provider is `thorswap` or `uniswap`  for a NanoS device ?

Comment (lpaquet-ledger) [reply apps/ledger-live-desktop/src/renderer/components/DeviceAction/index.tsx:343]: Yes, if the user is using a nano S and wants to swap with uniswap or Thorswap

Comment (jiyuzhuang) [reply apps/ledger-live-desktop/src/renderer/components/DeviceAction/index.tsx:343]: Then I think you need to update the conditions, the actual code will let all swap requests using `uniswap` pass no matter on which device.

Comment (lpaquet-ledger) [reply apps/ledger-live-desktop/src/renderer/components/DeviceAction/index.tsx:343]: ` (device?.modelId === "nanoS" && (request as SwapRequest)?.provider === "thorswap") ||
    (request as SwapRequest)?.provider === "uniswap"
  )`
  
  If it's Thorswap or Uniswap and you are on a Nano S you are returned a error drawer, I don't understand

Comment (jiyuzhuang) [reply apps/ledger-live-desktop/src/renderer/components/DeviceAction/index.tsx:343]: If the device is not NanoS, but the swap provider is `uniswap`, the code in the if-scope will still executed.

**Pull request closed without merging**